packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -814,6 +814,8 @@ export interface SdkPagingServiceMetadata<TServiceOperation extends SdkServiceOp
   continuationTokenParameterSegments?: SdkModelPropertyType[];
   /** Segments to indicate how to get continuation token value from response. */
   continuationTokenResponseSegments?: SdkModelPropertyType[];
+  /** Segments to indicate how to get page items from response. */
+  pageItemsSegments?: SdkModelPropertyType[];
```

Comment:
Should it be `required`?

Reply:
for azure, it might be undefined. sad. https://github.com/Azure/typespec-azure/blob/405af28b4b9b3c16aa7de4bcbbb704bd12408249/packages/typespec-azure-core/src/decorators.ts#L101